integration_tests/pom.xml

```diff
@@ -64,14 +64,6 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
-        <!-- use aggregator jar because accessing internal classes -->
-        <dependency>
-            <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark-aggregator_${scala.binary.version}</artifactId>
```

Comment:
This alone does not remove the dependency.  I think it's a step in the right direction, but rapids-4-spark-udf-examples and rapids-4-spark-tests are still here and both pull in the aggregator jar as a dependency.  Thus this will transitively depend on the aggregator jars, so the issue is still not fixed.  We would need to sever the connection to these other two modules (or update them so they no longer need the aggregator jars).

Reply:
I think we don't need to change it in this PR.
This PR is just to stop publishing the integration jar in release building.

Reply:
fixed